Makefile

```diff
@@ -126,6 +126,14 @@ check-license: $(MYGOBIN)/addlicense
 lint: $(MYGOBIN)/golangci-lint $(MYGOBIN)/goimports $(builtinplugins)
 	./hack/for-each-module.sh "make lint"
 
+.PHONY: apidiff
+apidiff: go-apidiff ## Run the go-apidiff to verify any API differences compared with origin/master
+	$(GOBIN)/go-apidiff master --compare-imports --print-compatible --repo-path=.
+
+.PHONY: go-apidiff
+go-apidiff:
+	go install github.com/joelanford/go-apidiff@v0.5.0
```

Comment:
I think Makefile uses the old `go-apidiff` binary. Could you update this?
https://github.com/kubernetes-sigs/kustomize/pull/5203/files#diff-be30c33d5f516eb4268cde1e472500e1c5005494ab6afa37ab3fc0b571804217R24

Reply:
I've updated the Makefile instead to use an updated version.